fixup-mountpoints

```diff
@@ -183,6 +183,16 @@ case "$DEVICE" in
             "$@"
         ;;
 
+    "a5y17lte")
+    sed -i \
+        -e 's platform/13540000.dwmmc0/by-name/SYSTEM mmcblk0p19 ' \
+        -e 's platform/13540000.dwmmc0/by-name/CACHE mmcblk0p20 ' \
+        -e 's platform/13540000.dwmmc0/by-name/USERDATA mmcblk0p25 ' \
+        -e 's platform/13540000.dwmmc0/by-name/EFS mmcblk0p3 ' \
+        -e 's platform/13540000.dwmmc0/by-name/HIDDEN mmcblk0p21 ' \
+        "$@"
+    ;;
+        
```

Comment:
Please remove the unused spaces in this line.

Reply:
done